utam-core/src/main/java/utam/core/selenium/element/ElementAdapter.java

```diff
@@ -163,7 +164,7 @@ public void scrollIntoView(ScrollOptions options) {
         driverAdapter.executeScript(SCROLL_TOP_VIA_JAVASCRIPT, getWebElement());
       }
       if (!isDisplayed()) {
-        throw new ElementNotVisibleException(SCROLL_INTO_VIEW_ERR);
+        throw new InvalidElementStateException(SCROLL_INTO_VIEW_ERR);
```

Comment:
We probably should throw something non selenium here to avoid fixing it in the future. Since our fluent wait now catches any exception, it should not affect any wrapped into waits invocations

Reply:
That's a very fair point. I don't know if it's required by this particular PR, but we can certainly throw a `UtamError`.

Reply:
This is now done via another PR.